LATEST

```diff
@@ -1,4 +1,5 @@
 ba6ba9019f0b3496fad68d936989fd9895cf3440 1.11.0-snapshot.20210217.6338.0.ba6ba901
+9d4b23229ae0f7ee51b424d8c90b42e8bd513446 1.10.1
```

Comment:
```suggestion
52c8caa76e6abc0e0f17a51562d11b7e97c3ab95 1.10.1
```
bumping to include https://github.com/digital-asset/daml/pull/8903#event-4353521209